src/css/base.scss

```diff
@@ -1783,6 +1787,15 @@ textarea {
     }
 }
 
+.icon-color-n7 {
+    path:nth-child(1) {
+        fill: none !important;
+    }
+    path:nth-child(2) {
+        fill: var(--N700);
```

Comment:
I think only applying fill to child 2 should work.

Reply:
Button css get applied on nth-child 1. Hence, applying none as important